frame/support/procedural/src/pallet/expand/storage.rs

```diff
@@ -381,6 +381,7 @@ pub fn expand_storages(def: &mut Def) -> proc_macro2::TokenStream {
 						#(#cfg_attrs)*
 						impl<#type_impl_gen> #pallet_ident<#type_use_gen> #completed_where_clause {
 							#( #docs )*
+							#[deprecated = "#[pallet::getter] is deprecated and will be unavailable in future versions."]
```

Comment:
Should we link to this PR / issue in this deprecation notice, then people can see how to remove the getter and replace it with a direct storage query, or the equivalent getter code.

Reply:
@muraca reminder in case you forgot about this comment